src/shared/containers/DashboardList.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { useAsyncEffect } from 'use-async-effect';
+import { Resource } from '@bbp/nexus-sdk';
+import { useNexusContext } from '@bbp/react-nexus';
+import TabList from '../components/Tabs/TabList';
+
+type Dashboard = {
+  dashboard: string;
+  view: string;
+};
+interface DashboardListProps {
+  dashboards: Dashboard[];
+  orgLabel: string;
+  projectLabel: string;
+}
+
+const DashboardList: React.FunctionComponent<DashboardListProps> = ({
+  dashboards,
+  orgLabel,
+  projectLabel,
+}) => {
+  const [dashBoards, setDashboards] = React.useState<Resource[]>([]);
+  const [selectedDashboard, setSelectedDashboard] = React.useState<Resource>();
+  const nexus = useNexusContext();
+  const selectDashboard = (id: string) => {
+    const dashboard = dashBoards.find(d => d['@id'] === id);
+    setSelectedDashboard(dashboard);
+  };
+  useAsyncEffect(async () => {
+    const dashboardList: Resource[] = [];
+    for (let i = 0; i < dashboards.length; i += 1) {
+      const dashboard = (await nexus.httpGet({
+        path: dashboards[i].dashboard,
```

Comment:
as mentioned before, we cannot do that because this relies on the fact the `@id` of this resource is also the `_selfUrl` which isn't (always) true.

Reply:
solved in aa742933fde51f14883f177bab765039e5bd3a66